### DISCUSSION TITLE
Can I use `tfcmt` for multiple Terraform states in the same PR?

### DISCUSSION THREAD

**Discussion by manudawber** [suzuki-shunsuke/tfcmt#675] [Announcements]
I currently have my infrastructure divided into multiple Terraform states (root modules) and want to use `tfcmt` to run `terraform plan` on each of these in a single pull request.

For example, I have a platform state and multiple application states. When a PR is opened/edited I want to run `tfcmt plan -patch=true -- terraform plan` for each state in a specific order. Ideally I would have one comment on the PR for each state, but at the moment I just have a single comment being patched each time the command is run.

I thought this may be configurable, e.g. through a `comment_id` setting, but haven't found one. Is this currently possible?

**Comment by suzuki-shunsuke** [comment]
To enable tfcmt to distinguish each comment, please set the variable `target`.
`target` should be unique per Terraform State.

https://suzuki-shunsuke.github.io/tfcmt/getting-started#monorepo-support-target-variable

```sh
tfcmt [-var "target:<target>"] plan -patch -- terraform plan
```

**Reply by manudawber** [reply]
I completely missed that in the documentation, thanks for your response! And also thanks for this tool, it's very useful.
